### ISSUE TITLE
Setting className to custom tag fails

### ISSUE THREAD

**Issue by unclechu** [purescript-contrib/purescript-react#134]
purescript-react is v5.1.0

When I declare my own tag name by this command:
```purescript
createElementTagName "my-own-tag-name" [className "container"]
```

And then when its rendered I got this tag in DOM tree but without `class` attribute and I see this warning in log:
```text
Warning: Invalid attribute name: `0`
warning.js:33
```

**Comment by unclechu** [comment]
Maybe I misunderstood something? I just used
```purescript
mkDOM (IsDynamic false) "my-own-tag-name" [class "container"]
```
instead and it works.

**Comment by unclechu** [comment]
I think I get it, `createElementTagName` is free to put anything as props, so I put array when I supposed to put a record, am I right?

**Comment by ethul** [comment]
Yes, you're correct. The props would have to be a record in this case.
